ntp.go

```diff
@@ -20,17 +20,12 @@ import (
 	"golang.org/x/net/ipv4"
 )
 
-type mode uint8
-
 const (
-	reserved mode = 0 + iota
-	symmetricActive
-	symmetricPassive
-	client
-	server
-	broadcast
-	controlMessage
-	reservedPrivate
+	// MaxStratum is the largest allowable NTP stratum value.
+	MaxStratum = 16
+
+	// MaxRootDistance is the largest possible root distance (in seconds).
+	MaxRootDistance = 2.96608
```

Comment:
Is it intentional to have unused constant like that?

Reply:
no, this should be removed.  I was originally going to add it to Validate, but decided against it.